src/dumpast.mli

```diff
@@ -0,0 +1,9 @@
+val transform : string -> Ppx_utils.kind -> Jsonoo.t
+
+val from_structure : Parsetree.structure -> Jsonoo.t
+
+val reparse :
+  Parsetree.structure_item list -> Parsetree.structure_item list -> Jsonoo.t
+
+val reparse_signature :
+  Parsetree.signature_item list -> Parsetree.signature_item list -> Jsonoo.t
```

Comment:
Could you move this and other ppx related modules to a sub library.

Reply:
Done in 59f461ea9985c9a23becf3b34f63d1e3e38ad7cd